tools/format_python_tools.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+VENV_DIR="pyformat"
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+cd "$SCRIPTPATH"
+
+if [ "${CIRCLECI}" != "true" ]; then
+
+  if [[ "$VIRTUAL_ENV" == "" ]]; then
```

Comment:
FWIW, this is now a pattern across all of the Envoy code base that uses virtualenv; would be rad to refactor, but not necessary.

Reply:
https://github.com/envoyproxy/envoy/pull/4951